apps/swirl-docs/src/pages/api-docs/[apiSpec].tsx

```diff
@@ -0,0 +1,296 @@
+            ).map((parameter) => {
+              const prop = parametersObject[parameter] as SchemaObject;
+
+              return {
+                name: String(parameter),
+                type: String(prop.type) || "",
+                description: prop.description || "",
+                required: requiredParams.includes(parameter),
+              };
+            });
+            return { title: label, type, parameters: parameters };
+          }
+          return { title: label, type, parameters: [] };
+        }),
+        request,
+        responseExamples: examples,
+      };
+    }),
+  };
+
+  return apiSpec;
+}
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  const specs = createStaticPathsForSpecs();
+
+  return {
+    fallback: false,
+    paths: specs,
+  };
+};
+
+export const getStaticProps: GetStaticProps = async (context) => {
+  if (!context.params || !("apiSpec" in context.params)) {
+    return {
+      notFound: true,
+    };
+  }
+
+  const { apiSpec } = context.params;
+  const document = await getSpecData(apiSpec as string);
+
+  return {
+    props: {
+      document,
+    },
+  };
+};
+
+export default function Document({ document }: { document: ApiDocumentation }) {
+  return (
+    <>
+      <Head>
+        <title>{`API | ${document.title}`}</title>
+      </Head>
+      <DocumentationLayout
+        data={{
+          mdxContent: {
+            document: document.description,
+            components: {
+              h1: (props) => <h2 className="text-2xl font-bold" {...props} />,
+              h2: (props) => <h3 className="text-xl font-bold" {...props} />,
+              a: (props) => (
+                <span className="inline-flex items-center text-interactive-primary-default">
+                  <a {...props} />
+                  <SwirlIconOpenInNew className="ml-1" size={20} />
+                </span>
+              ),
+              ul: (props) => (
+                <ul className="mb-4 leading-line-height-xl" {...props} />
+              ),
+              p: (props) => (
+                <p className="text-base leading-line-height-xl">
+                  {props.children}
+                </p>
+              ),
+              code: (props) => (
+                <code
+                  className="bg-gray-100 rounded-md p-1 text-sm font-font-family-code"
+                  {...props}
+                />
+              ),
+            },
+          },
+          frontMatter: {
+            title: document.title,
+            description: document.shortDescription,
+            examples: [],
+          },
+          navigationLinks: apiDocsNavItems,
+        }}
+        disableToc
+        header={<DocumentationLayout.Header className="col-span-2" />}
+        content={
+          <>
+            <DocumentationLayout.MDX />
+            <div className="mt-20">
+              {document.endpoints?.map((endpoint, index) => {
+                return (
+                  <>
```

Comment:
**issue:** Missing a key. You could use `<article aria-labelledby={endpoint.path.split("#")[1]}>` here, in addition.